services/provider/server/server.go

```diff
@@ -117,6 +117,10 @@ func (s *OCSProviderServer) OnboardConsumer(ctx context.Context, req *pb.Onboard
 			return nil, status.Errorf(codes.Internal, "failed to create storageConsumer %q. %v", req.ConsumerName, err)
 		}
 
+		if err == errTicketInvalidStorageQuota {
+			return nil, status.Errorf(codes.InvalidArgument, "onboarding ticket with quota is not valid %q", req.OnboardingTicket)
```

Comment:
Use `invalid` instead of "not valid" and use "contains" instead of "with" 
```suggestion
			return nil, status.Errorf(codes.InvalidArgument, "onboarding ticket contains an invalid storage quota value" %q", req.OnboardingTicket)
```